packages/labs/examples/timezoneSelectExtendedExample.tsx

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as moment from "moment-timezone";
+import * as React from "react";
+
+import { Classes, Icon, Switch, Tag } from "@blueprintjs/core";
+import { DatePicker, TimePicker, TimePickerPrecision } from "@blueprintjs/datetime";
+import { BaseExample, handleBooleanChange, handleStringChange } from "@blueprintjs/docs";
+
+import { ITimezoneSelectTargetRenderer, TimezoneDisplayFormat, TimezoneSelect } from "../src";
+
+export interface ITimezoneSelectExtendedExampleState {
+    date?: Date;
+    time?: Date;
+    timezone?: string;
+    dateTime?: Date;
+    targetDisplayFormat?: TimezoneDisplayFormat;
+    disabled?: boolean;
+    showLocalTimezone?: boolean;
+    useDefault?: boolean;
+    defaultToUserTimezoneGuess?: boolean;
+    useCustomRenderer?: boolean;
+}
+
+const EXAMPLE_DEFAULT_TIMEZONE = "Pacific/Honolulu";
+
+export class TimezoneSelectExtendedExample extends BaseExample<ITimezoneSelectExtendedExampleState> {
```

Comment:
i'm not sure this example is valuable. it doesn't show much beyond the basic example. would be fine moving props switches to basic example and killing this (can always add later).